src/screens/Home/blankSlate.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BlankSlate = () => {
+  return (
+    <div className="blank-slate">
+      <span
+        role="img"
+        aria-label="sunglass-emoji"
+        className="blank-slate__emoji"
+      >
+        😎
+      </span>
+      <h3 className="blank-slate__message">You’ve completed all the survey.</h3>
```

Comment:
@rafayet-monon I don't think we should use `h3` for making such a huge text for this situation 🤔
 I think we will use `h-` tag for headline stuff.
So this block I think it should be just `p` or `div`, what do you think?